hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonFileInputFormat.java

```diff
@@ -279,7 +279,8 @@ public void setAllColumnProjectionIfNotConfigured(JobContext job, CarbonTable ca
   private String[] getDeleteDeltaFiles(String segmentFilePath, List<String> allDeleteDeltaFiles) {
     List<String> deleteDeltaFiles = new ArrayList<>();
     String segmentFileName = null;
-    String[] pathElements = segmentFilePath.split(Pattern.quote(File.separator));
+    segmentFilePath = segmentFilePath.replace("\\", "/");
```

Comment:
Use WINDOWS_FILE_SEPARATOR and FILE_SEPARATOR at place of string

Reply:
Done